src/components/services/PiHole.vue

```diff
@@ -37,6 +37,9 @@ export default {
     ads_percentage_today: 0,
   }),
   computed: {
+    apiQuery() {
+      return this.item.apikey ? "?summaryRaw&auth=${this.item.apikey}" : "";
```

Comment:
I believe this needs to be:

```js
return this.item.apikey ? `?summaryRaw&auth=${this.item.apikey}` : "";
```

Reply:
@tibbe I was looking at other files and @bastienwirtz is usually adding backticks to the function call, something like:

```
const result = await this.fetch(`/api.php${this.apiQuery()}`)
```

Reply:
I believe you need the backticks every time you do variable interpolation (i.e. every time you mention `${..}`). which happens on this line. Perhaps it's possible to delay the interpolation but that seems error prone (e.g. the interpolated expression might not be in scope at the later point).

Reply:
@tibbe thank you for the info, not experienced with due language, Python head here. :) Basically, I should add the backticks in both locations, right? If yes, I'll push the changes now, let me know please.

Reply:
Yes I believe they need to be in both places and contain the entire string (like I did I my examples).

Reply:
@tibbe, I pushed the change.